backend/src/schema/resolvers/registration.js

```diff
@@ -40,7 +40,7 @@ export default {
       `,
         { nonce, email },
       )
-      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('Email'))
+      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('EmailAddress'))
       if (!emailAddress) throw new UserInputError('Invalid email or nonce')
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-17T16:19:43Z" title="Tuesday, December 17th 2019, 5:19:43 pm +01:00">Dec 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,7 +40,7 @@ export default {
       `,
         { nonce, email },
       )
-      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('Email'))
+      const emailAddress = await neode.hydrateFirst(result, 'email', neode.model('EmailAddress'))
       if (!emailAddress) throw new UserInputError('Invalid email or nonce')
```


what would we have to do to remove `neode` from production code?
This is one of the only remaining resolvers which uses it...

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-17T18:03:10Z" title="Tuesday, December 17th 2019, 7:03:10 pm +01:00">Dec 17, 2019</time>_

---


The fact that @johnymontana considers using neode in neo4j-graphql-js is making me unsure about if we should try to refactor neode out of our production code.